tests/rule_based_profiler/alice_user_workflow_verbose_profiler_config.yml

```diff
@@ -82,38 +82,34 @@ rules:
         column: $domain.domain_kwargs.column
         min_value: $variables.min_timestamp
         max_value: $variables.min_timestamp
-      # TODO: These additional kwargs are not being handled correctly
-      #   meta:
-      #     notes:
-      #       format: markdown
-      #       content:
-      #         - ### This expectation confirms no events occur before tracking started **2004-10-19 10:23:54**
-      # TODO: This expectation should be replaced with the expectation below that references a parameter
-      # TODO: from a specific domain
+        meta:
+          notes:
+            format: markdown
+            content:
+              - "### This expectation confirms no events occur before tracking started **2004-10-19 10:23:54**"
+      # TODO: This expectation should be replaced with the expectation below that references a parameter from a specific domain
       - expectation_type: expect_column_max_to_be_between
         class_name: DefaultExpectationConfigurationBuilder
         module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
         column: $domain.domain_kwargs.column
         min_value: $variables.min_timestamp
         max_value: $parameter.my_max_event_ts.column.max
-        # TODO: These additional kwargs are not being handled correctly
-#        meta:
-#         notes:
-#           format: markdown
-#           content:
-#             - ### This expectation should be replaced with the below expectation
+        meta:
+          notes:
+            format: markdown
+            content:
+              - "### This expectation should be replaced with the below expectation"
         # TODO: This expectation needs to access a parameter for a set domain (event_ts)
         # TODO: to confirm that the event_ts contains the latest timestamp of all domains
 #      - expectation_type: expect_column_max_to_be_between
 #        class_name: DefaultExpectationConfigurationBuilder
 #        module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
 #        column: $domain.domain_kwargs.column
 #        min_value: $variables.min_timestamp
-        # TODO: How to reference the event_ts domain?
+        # TODO: How to reference the event_ts domain?<Alex>ALEX -- Anthony: ParameterBuilder has access to domain name; hence, fully_qualified_parameter_name can contain column_name (from domain_kwargs).  Would this work?</Alex>
```

Comment:
I think so! We did something similar in the Bob config: https://github.com/great-expectations/great_expectations/blob/0efe6c261bfd11f45f104b6b35f987a058c50cff/tests/rule_based_profiler/bob_user_workflow_verbose_profiler_config.yml#L107-L113

Reply:
Just confirming - I believe this is another piece of functionality that relies on active_batch, or else we wouldn't know where to look for that column.

Reply:
Good point. This assumes we look for the domain during the currently processed batch. I'm assuming that is done in an outer loop with an inner loop processing each domain in the currently in-process batch.

Reply:
I think that makes sense. This is where I would think it would be possible to pass in other domain_kwargs, like `table` or `batch_id`, but where if we don't pass these in, we assume them from the active_batch. That's what I would expect, but I'm having trouble imagining if that is how this would actually work.